lib/Elastica/Response.php

```diff
@@ -78,14 +78,56 @@ public function __construct($responseString, $responseStatus = null)
      */
     public function getError()
     {
-        $message = '';
+        $error = $this->getFullError();
+
+        if (!$error) {
+            return '';
+        }
+
+        if (is_string($error)) {
+            return $error;
+        }
+
+        if (isset($error['root_cause'][0])) {
+            $error = $error['root_cause'][0];
```

Comment:
Do you really want to overwrite the error varialbe here? what if there are multiple root_causes?